### ISSUE TITLE
ERROR: failed to create the CAROOT: mkdir /mnt/c/Users/YOUR_WINDOWS_USERNAME: permission denied - on Ubuntu 20.04 on WSL2

### ISSUE THREAD

**Issue by therobyouknow** [FiloSottile/mkcert#270]
If I run 
`mkcert -install`

I get:

`ERROR: failed to create the CAROOT: mkdir /mnt/c/Users/YOUR_WINDOWS_USERNAME: permission denied`

How can I get mkcert -install to run correctly please?

In trying to solve the issue, I did:
```
$ which mkcert
/home/linuxbrew/.linuxbrew/bin/mkcert
```

And then tried

$ sudo su root
$ /home/linuxbrew/.linuxbrew/bin/mkcert

which results in Creating the local CA at "/root/.local/share/mkcert"

But I don't think this is the right place for the CA, it should be somewhere in the home folder of my non-root user.

Related issue - I'm doing mkcert -install as part of setting up ddev on Ubuntu 20.04 wsl2
https://github.com/drud/ddev/issues/2311

**Comment by therobyouknow** [comment]
This looks like the solution:

In my` .profile` file there was

`export CAROOT=/mnt/c/Users/YOUR_WINDOWS_USERNAME/AppData/Local/mkcert`

I changed this to

`export CAROOT=/mnt/c/Users/Rob/AppData/Local/mkcert`

And then closed my (only) shell for Ubuntu 20.04 and re-opened it, so that it would re-run .profile

And it looks like it's working.

```
$ mkcert -install
Using the local CA at "/mnt/c/Users/Rob/AppData/Local/mkcert" ✨
Sudo password:
The local CA is now installed in the system trust store! ⚡️
```

Also posted as comment on https://github.com/drud/ddev/issues/2311

Will close if I now get https for local site URL on ddev start

I do wonder how `export CAROOT=/mnt/c/Users/YOUR_WINDOWS_USERNAME/AppData/Local/mkcert` was used, not sure if any docs mention it. It's bonkers! But at least I think I now have a solution.

**Comment by therobyouknow** [comment]
Solved PEBKAC (I me!) see my write up here: https://github.com/drud/ddev/issues/2311#issuecomment-643993480

PEBKAC definition: https://www.computerhope.com/jargon/p/pebkac.htm#:~:text=Short%20for%20problem%20exists%20between,software%20or%20the%20device%20itself.
